prime-dotnet-webapi/Services/EmailDocumentsService.cs

```diff
@@ -71,6 +71,37 @@ public async Task SaveSiteRegistrationReview(int siteId, Pdf pdf)
             await _context.SaveChangesAsync();
         }
 
+        public async Task<Pdf> GenerateHealthAuthorityRegistrationReviewAttachmentAsync(int healthAuthoritySiteId)
+        {
+            var model = await _context.HealthAuthoritySites
+                .Where(has => has.Id == healthAuthoritySiteId)
+                .Select(has => new HealthAuthoritySiteSubmissionViewModel
+                {
+                    SiteName = has.SiteName,
+                    SiteAddress = has.PhysicalAddress,
+                    AuthorizedUserName = $"{has.AuthorizedUser.Party.FirstName} {has.AuthorizedUser.Party.LastName}",
+                    HealthAuthorityName = has.HealthAuthorityOrganization.Name,
+                    PEC = has.PEC,
+                    CareType = has.HealthAuthorityCareType.CareType,
```

Comment:
Not sure what is the desired naming, should we be calling this CareSettingType?

Reply:
Just bringing this up because of a conversation in the last sprint review

Reply:
Yeah I remember they mentioned something about that, they haven't updated us, and if I'm not mistaken, I think Sarah said something about HAs using care type. I picked the naming from the FE and other models under HA

Reply:
In Admin:
![image](https://user-images.githubusercontent.com/39168456/141209470-7d80b5f8-e622-4eba-8919-65748a656660.png)

In HA site enrollment:
![image](https://user-images.githubusercontent.com/39168456/141209566-48d1054e-c70c-4062-a2b2-e98c9db83f68.png)

So it is "Care Type"